lib/node/finalize.js

```diff
@@ -60,5 +62,8 @@ export default function getFinalizeCallback(node, state) {
 
     // Dispatch an event on the node once rendering has completed.
     node.dispatchEvent(new CustomEvent('renderComplete'));
+
+    // Call the remaining middleware with a completion Date.
+    remainingMiddleware.forEach(f => f(new Date()));
```

Comment:
`performance.now()` is higher precision than `new Date()`. I'd also be curious to see if it's any faster.